src/main/java/com/conveyal/gtfs/loader/EntityPopulator.java

```diff
@@ -214,9 +243,65 @@
         stopTime.continuous_drop_off = getIntIfPresent   (result, "continuous_drop_off", columnForName);
         stopTime.timepoint           = getIntIfPresent   (result, "timepoint", columnForName);
         stopTime.shape_dist_traveled = getDoubleIfPresent(result, "shape_dist_traveled", columnForName);
+        stopTime.pickup_booking_rule_id = getStringIfPresent(result, "pickup_booking_rule_id", columnForName);
+        stopTime.drop_off_booking_rule_id = getStringIfPresent(result, "drop_off_booking_rule_id", columnForName);
+        stopTime.start_pickup_dropoff_window = getIntIfPresent(result, "start_pickup_dropoff_window", columnForName);
+        stopTime.end_pickup_dropoff_window = getIntIfPresent(result, "end_pickup_dropoff_window", columnForName);
+        stopTime.mean_duration_factor = getDoubleIfPresent(result, "mean_duration_factor", columnForName);
+        stopTime.mean_duration_offset = getDoubleIfPresent(result, "mean_duration_offset", columnForName);
+        stopTime.safe_duration_factor = getDoubleIfPresent(result, "safe_duration_factor", columnForName);
+        stopTime.safe_duration_offset = getDoubleIfPresent(result, "safe_duration_offset", columnForName);
         return stopTime;
     };
 
+    EntityPopulator<BookingRule> BOOKING_RULE = (result, columnForName) -> {
+        BookingRule bookingRule = new BookingRule();
+        bookingRule.booking_rule_id = getStringIfPresent(result, "bookingRule", columnForName);
+        bookingRule.booking_type = getIntIfPresent(result, "booking_type", columnForName);
+        bookingRule.prior_notice_duration_min = getIntIfPresent(result, "prior_notice_duration_min", columnForName);
+        bookingRule.prior_notice_duration_max = getIntIfPresent(result, "prior_notice_duration_max", columnForName);
+        bookingRule.prior_notice_last_day = getIntIfPresent(result, "prior_notice_last_day", columnForName);
+        bookingRule.prior_notice_last_time = getIntIfPresent(result, "prior_notice_last_time", columnForName);
+        bookingRule.prior_notice_start_day = getIntIfPresent(result, "prior_notice_start_day", columnForName);
+        bookingRule.prior_notice_start_time = getIntIfPresent(result, "prior_notice_start_time", columnForName);
```

Comment:
These are correct, but see my comment about changing to the correct type in the schema.

Reply:
Thanks for checking.